### PR TITLE
Minor fixes.

### PR DESCRIPTION
**Pull request by ygorshenin**
@mgsergio PTAL

### CODE REVIEW

File: openlr/openlr_stat/openlr_stat.cpp [hunk 1]
```diff
@@ -27,35 +33,14 @@ void LoadIndex(Index & index)
   auto const latestVersion = numeric_limits<uint64_t>::max();
   FindAllLocalMapsInDirectoryAndCleanup(dir, 0 /* version */, latestVersion, localFiles);
 
-  Platform::TFilesWithType fwts;
```
Comment (mgsergio) [inline openlr/openlr_stat/openlr_stat.cpp]: Will this work with structure like this
- data
  - 161010
    - mwm1
    - mwm2
    - ...
      ?

Comment (ygorshenin) [reply openlr/openlr_stat/openlr_stat.cpp]: Fixed. Works now.

**Comment by ygorshenin** [comment]
PTAL

**Comment by mgsergio** [comment]
LGTM

**Pull request merged**